org.kicad.KiCad.Library.Templates.yml

```diff
@@ -0,0 +1,40 @@
+id: org.kicad.KiCad.Library.Templates
+branch: '5.1'
```

Comment:
It should target `stable`; we don't "support" multiple branches for apps and apps extensions.

Reply:
Ok, didn't know that before, and it has worked OK so far for org.kicad.KiCad.Packages3D. Will fix it later.

Reply:
@barthalion Could you please elaborate why flathub doesn't "support" branches? The extension endpoint definition in  `org.kicad.KiCad` was using branch `5.1` before, and this was not considered an issue, when I submitted `org.kicad.KiCad.Library.Packages3D` with this setting.

Given this, changing everything to move to `stable` would now mean to alter both the extension(s) and the main app.

For reference, here is the current definition of the extension mountpoint from `org.kicad.KiCad`:
```
add-extensions:
  org.kicad.KiCad.Library:
    version: '5.1'
    directory: extensions/Library
    autodelete: true
    no-autodownload: true
    subdirectories: true
    merge-dirs: share/kicad/template
```

Reply:
Applications target only the `stable` branch, so there is no reason to version an extension if it's not going to be used by any other application. This usually passes reviews because there's no linter yet so the process is prone to oversights.

Reply:
Ok, the I will move all of the proposed extensions, and the mount point to `stable` instead.